content/en/docs/components/trainer/user-guides/builtin-trainer/torchtune.md

```diff
@@ -0,0 +1,279 @@
++++
+title = "TorchTune LLM Trainer"
+description = "How to use TorchTune LLM Trainer in BuiltinTrainer"
+weight = 20
++++
+
+**TorchTune LLM Trainer** is one of the supported BuiltinTrainers in Kubeflow SDK. It leverages TorchTune to streamline and simplify LLM fine-tuning lifecycle on Kubernetes, while providing a simple yet efficient Python API for data scientists.
+
+The supported model list can be seen in [this directory](https://github.com/kubeflow/trainer/tree/master/manifests/base/runtimes/torchtune). It's worth noticing that we do not support multi-node fine-tuning for now. Only one Pod will be launched for TorchTune LLM Trainer.
+
+In this page, we'll show a simple example of fine-tuning LLMs with TorchTune LLM Trainer in Kubeflow SDK. If you want to learn more about TorchTune LLM Trainer, please refer to [KEP-2401](https://github.com/kubeflow/trainer/tree/master/docs/proposals/2401-llm-trainer-v2) in Kubeflow Trainer.
+
+## Prerequisites
+
+Ensure that you have access to a Kubernetes cluster with Kubeflow Trainer
+control plane installed. If it is not set up yet, follow
+[the installation guide](/docs/components/trainer/operator-guides/installation) to quickly deploy
+Kubeflow Trainer.
+
+```Python
+# List all available Kubeflow Training Runtimes.
+from kubeflow.trainer import *
+
+client = TrainerClient()
+for runtime in client.list_runtimes():
+    print(runtime)
+```
+
+### Installing the Kubeflow Python SDK
+
+Install the latest Kubeflow Python SDK version directly from the source repository:
+
+```bash
+pip install git+https://github.com/kubeflow/sdk.git@main#subdirectory=python
+```
+
+## Use TorchTune LLM Trainer in BuiltinTrainer
+
+### Create PVCs for Models and Datasets
+
+{{% alert title="Note" color="info" %}}
+Currently, we do not support automatically orchestrate the volume claim.
+
+To follow up with this problem, please refer to [this issue](https://github.com/kubeflow/trainer/issues/2630).
+{{% /alert %}}
+
+We need to manually create PVCs for each models we want to fine-tune. Please note that **the PVC name must be equal to the ClusterTrainingRuntime name**. In this example, it's `torchtune-llama3.2-1b`.
+
+```Python
+# Create a PersistentVolumeClaim for the TorchTune Llama 3.2 1B model.
+client.core_api.create_namespaced_persistent_volume_claim(
+  namespace="default",
+  body=client.V1PersistentVolumeClaim(
+    api_version="v1",
+    kind="PersistentVolumeClaim",
+    metadata=client.V1ObjectMeta(name="torchtune-llama3.2-1b"),
+    spec=client.V1PersistentVolumeClaimSpec(
+      access_modes=["ReadWriteOnce"],
+      resources=client.V1ResourceRequirements(
+        requests={"storage": "20Gi"}
+      ),
+    ),
+  ),
+)
+```
+
+### Use TorchTune LLM Trainer with `train()` API
```

Comment:
Let's not use the `<code>` block in the titles, since the website renders it blurry